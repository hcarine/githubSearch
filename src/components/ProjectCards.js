import React from 'react';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import Typography from '@material-ui/core/Typography';

const ProjectCards = (props) => {
    return props.company.map(project=>
        (
        <Card>
            <CardContent>
              <Typography variant="h5" component="h2">
                {project.name}
              </Typography>
              <Typography color="textSecondary">
                { project.language}x
              </Typography>
              <Typography variant="body2" component="p">
                { project.stars}
              </Typography>
            </CardContent>
          </Card>
        ))
};

export default ProjectCards;