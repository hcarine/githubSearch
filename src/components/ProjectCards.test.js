import React from 'react';
import { mount } from 'enzyme';
import ProjectCards from './ProjectCards';

describe('As a user i want to see my many converted', () => {
    const props={
        projects:[ {
            name: 'test',
            languege: 'stranger thing',
            stargazers_count: 5
        }]
    }
   
const wrapper = mount(<ProjectCards {...props}/>);
    it('Check if the component is mounted', () => {
        expect(wrapper).toMatchSnapshot();
    });
})