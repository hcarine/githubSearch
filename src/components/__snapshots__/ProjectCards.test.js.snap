// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As a user i want to see my many converted Check if the component is mounted 1`] = `ReactWrapper {}`;
